### DISCUSSION TITLE
Functions in MQTT expression?

### DISCUSSION THREAD

**Discussion by JabeBRD** [meatpiHQ/wican-fw#148] [Q&A] [answered]
What functions does Wican support in MQTT json expression?
Is it possible to round output value?
Is it possible to output multiple bytes? Like B1, B2, B3.

**Comment by meatpiHQ** [comment, marked as answer]
If you want 3 bytes you should use V instead of B, and set the 

start bit: 8
Bit length: 24

No you cannot round.
